samples/full-bundle/README.md

```diff
@@ -1,15 +1,145 @@
-# Sample
+# Sample -  Getting Started with Web Chat CDN
 
-A simple web page with a maximized and full-featured Web Chat.
+> This is a great sample for first-time Web Chat users.
 
-# How to run
+## Description
+A simple web page with a maximized and full-featured Web Chat embed from a CDN. This includes Adaptive Cards, Cognitive Services, and Markdown-It dependencies. 
 
-- Run `npx serve`
+
+# Test out the hosted sample
+- [Try out MockBot](https://microsoft.github.io/BotFramework-WebChat/full-bundle)
+
+# How to run locally
+
+- Fork and clone this repo
+- Navigate to `/Your-Local-WebChat/samples/full-bundle` in command line
+- Run `npx serve` in the full-bundle directory
 - Browse to [http://localhost:5000/](http://localhost:5000/)
 
 # Things to try out
+- Type `help`: you should see a full list of MockBot features
+- Type `markdown`: you should see the sample of Markdown
+- Type `card weather`: you should see a weather card built using Adaptive Cards
+- Type `layout carousel`: you should see a carousel of cards
+   - Resize the window and see how the carousel changes size
+
+# Code
+
+> Jump to [completed code](#completed-code) to see the end-result `index.html`.
+
+## Getting started
+### Goals of this bot
+This code features the minimal scripting the bot needs to host a full-featured Web Chat. 
+The `index.html` page has two main goals.
+- To import the Web Chat full bundle CDN script
+- To render Web Chat 
+
+ We'll start by adding the CDN to the head of a blank `index.html` template.
+```diff
+…
+<head>
++ <script src="https://cdn.botframework.com/botframework-webchat/preview/botchat.js"></script>
```

Comment:
Is this link going to change when webchat goes out of preview? If so, it might be good to set an issue/work item to go back over the samples with links to the CDN and update them after or right before preview ends.